curriculum/challenges/english/25-front-end-development/quiz-recursion/66edd43cded6bff30944b676.md

```diff
@@ -17,439 +17,507 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is recursion in programming?
 
 #### --distractors--
 
-Placeholder distractor 1
+A programming technique where a function calls itself.
 
 ---
 
-Placeholder distractor 2
+A method for sorting arrays.
 
 ---
 
-Placeholder distractor 3
+A way to optimize loops.
 
 #### --answer--
 
-Placeholder answer
+A programming technique where a function calls itself.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be the output of the following function call: `fibonacci(5)`?
+
+```cpp
+int fibonacci(int n) {
+    if (n <= 1) return n;
+    return fibonacci(n - 1) + fibonacci(n - 2);
+}
+```
```

Comment:
please write your code in JavaScript

Reply:
This comment should not have been resolved. The issue is still present.